tests/transactional/check_phub.pm

```diff
@@ -0,0 +1,40 @@
+# Copyright 2023 SUSE LLC
+# SPDX-License-Identifier: GPL-2.0-or-later
+#
+# Summary: Test packagehub repo on sle micro
+#
+# 1. Check if PackageHub extention is availble on sle micro
+# 2. Activate PackageHub extention
+# 3. Install a package from PackageHub repo
+#
+# Maintainer: qac <qa-c@suse.de>
+
+package check_phub;
+use base 'consoletest';
+use strict;
+use warnings;
+use testapi;
+use version_utils 'is_sle_micro';
+use transactional;
+
+sub run {
+    select_console 'root-console';
+    my $out = script_output("transactional-update --quiet register --list-extensions");
+    if ($out =~ m#PackageHub/(.*)/#) {
+        trup_call("register -p PackageHub/$1/" . get_var('ARCH'));
```

Comment:
Please do this step only if the phub is not registered. In the DVD test case I created, the phub extension is registered at yast installation time.